CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.1.4 - 2022-10-01
```

Comment:
Hate to be a stickler, but shouldn't this be 0.2.0, since it's more adding functionality to the bot as opposed to a patch? Avg might help draw the line here though.

Reply:
¯\_(ツ)_/¯

Makes sense to me